extensions/Worker.Extensions.EventHubs/EventHubTriggerAttribute.cs

```diff
@@ -5,15 +5,16 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
-    public sealed class EventHubTriggerAttribute : TriggerBindingAttribute
+    public sealed class EventHubTriggerAttribute : TriggerBindingAttribute, IBatchedInput
     {
         /// <summary>
         /// Create an instance of this attribute.
         /// </summary>
         /// <param name="eventHubName">Event hub to listen on for messages. </param>
-        public EventHubTriggerAttribute(string eventHubName)
+        public EventHubTriggerAttribute(string eventHubName, bool isBatched = true)
```

Comment:
This is required for default - constructor name must match property name

Reply:
If I understand correctly, I am guessing you had to add this as right now we only get values that were defined during the instantiation (constructor args, and props) through `CustomAttributes`, and so if there is a static default value not declared here or during instantiation, it's not recorded.

I wonder if instead of doing it here, we should special case this one in sorts that -- if `IBatchedInput` is implemented, we go and look for the property `IsBatched` from the FunctionMetadataGenerator to get the default value set in the property.
I was looking through, out of curiosity (when I needed that), and looks like we could get constructor instructions to see the default value of static types. There might be a much simpler way. But here's how it looked like -- 
```csharp
CustomAttribute.AttributeType.Resolve().Methods.FirstOrDefault(m => m.Name == '.ctor').Body.Instructions.FirstOrDefault(i.Operand == <arg-type>).Previous.Operand
```

Reply:
We dug through this and ran into some trouble with looking through the instructions for "body". Instead, for Attributes that require a default value, we're adding a new attribute and checking for that!